cedar-policy-validator/src/rbac.rs

```diff
@@ -1149,6 +1150,48 @@ mod test {
         (principal_euid, action_euid, resource_euid, schema)
     }
 
+    fn assert_validate_policy_succeeds(schema: ValidatorSchema, policy: &Template) {
+        let validate = Validator::new(schema);
+        assert!(
+            validate
+                .validate_policy(&policy, ValidationMode::default())
+                .0
```

Comment:
Should this also check `.1` for warnings? Not strictly necessary for validation success, but it should be the case for the tests